app/src/lib/components/Carousel/Carousel.tsx

```diff
@@ -63,26 +101,74 @@ Carousel.defaultProps = {
   slideToShow: 1,
   isArrowShow: true,
   isAutoplay: false,
+  arrowLocation: 'mid-side',
+  prevArrowIcon: <FaArrowCircleLeft />,
+  nextArrowIcon: <FaArrowCircleRight />,
 };
 
 const Wrapper = styled.div<{
+  arrowLocation: 'bottom' | 'mid-side' | 'top' | 'bottom-side' | 'top-side';
+  width?: string;
+  padding?: string;
+}>`
+  width: ${(props) => props.width || '100%'};
+  padding: ${(props) => props.padding || 0};
+  position: relative;
+
```

Comment:
width props받으니까 컨텐츠가 left기준으로 위치해서 중앙정렬 들어가면 좋을거 같아요
```suggestion
margin : 0 auto;
```suggestion

```
```